docs/blog/2016-9-28.md

```diff
@@ -1,3 +1,5 @@
+###...
+
 # 0.7 Release
```

Comment:
Let's remove everything from this file related to 0.x releases. Let's retain only External blog entries section.

Blogs section should contain just a single page that provides pointers to VMware as well as external blogs. https://github.com/vmware/docker-volume-vsphere/blob/master/README.md#blogs contains the updated list.